### PR TITLE
Create box_it_script and add solution.js

### PR DESCRIPTION
**Pull request by amirsalaar**
First homework

### CODE REVIEW

File: box_it_script/index.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
+#!/home/amirsalar/.nvm/versions/node/v11.10.1/bin/node
+
+const inputs = process.argv.slice(2);
+
+// To read csv file and store its content
+function csvChecker() {
+    if (inputs[0]) {
+        if (inputs[0].includes('.csv')) {
+            const fs = require('fs');
+            const contents = fs.readFileSync(inputs[0], 'utf8');
```
Comment (QuickStyles) [inline box_it_script/index.js]: I would rename this variable to something more specific. like `fileContents` or `csvContents`

File: box_it_script/index.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+    }
+}
+
+
+if (!csvChecker()) {
+    console.log(boxIt(inputs));
+} else {
+    console.log(boxCSV(inputs));
+}
```
Comment (QuickStyles) [inline box_it_script/index.js]: standard practice is to leave a new line at the end of each file

File: box_it_script/index.js [hunk 3]
```diff
@@ -0,0 +1,121 @@
+#!/home/amirsalar/.nvm/versions/node/v11.10.1/bin/node
+
+const inputs = process.argv.slice(2);
+
+// To read csv file and store its content
+function csvChecker() {
```
[the remark this answers is not in the document]
Comment (amirsalaar) [reply box_it_script/index.js]: Implemented!

File: box_it_script/index.js (hunk 1 above)
Comment (amirsalaar) [reply box_it_script/index.js]: changed it to cliInput
because sometimes we just have strings passed to the script as arguments, not CSV files

**Pull request merged**